example/App.tsx

```diff
@@ -169,7 +180,10 @@ export default function App() {
   return (
     <GestureHandlerRootView>
       <NavigationContainer>
```

Comment:
Can you add `<StatusBar barStyle="dark-content" />` also (I have no idea why the status bar is light in my case, but it's bothering me very much 😅)?

Reply:
done